buildbot/configure.py

```diff
@@ -32,16 +32,16 @@ def do_configure(args):
 
     icd_loader_lib = os.path.join(icd_loader_lib, "libOpenCL.so" if platform.system() == 'Linux' else "OpenCL.lib")
 
+    # replace not append, so ARM ^ X86
```

Comment:
What is the reason to drop X86 target? Save on compilation time?

Reply:
Yes, precisely that.  Many ARM systems are not well-suited for compiling LLVM, and the likelihood that someone will want to cross-compile for x86 on an ARM device seems rather low.  Building on a Raspberry Pi 4 takes approximately 10 hours as it is.  Eventually, I'll try to generalize this but I figure anybody who is into cross-compilation can grab the CMake commands from the builtbot script and do it manually.